src/main/scala/extracells/part/PartFluidStorage.java

```diff
@@ -318,4 +321,66 @@ private boolean wasChanged() {
         }
         return !fluids.equals(fluidList);
     }
+
+    @Override
+    public NBTTagCompound transformPart(NBTTagCompound def) {
+        if (Integration.Mods.FLUIDCRAFT.isEnabled()) {
+            FluidCraft.replace(def, PartEnum.FLUIDSTORAGE);
+        }
+        return def;
+    }
+
+    private static int[] FILTER_MAP = null;
+    static {
+        if (Integration.Mods.FLUIDCRAFT.isEnabled()) {
+            // spotless:off
+			FILTER_MAP = new int[] {
+				0,  9, 18, 27, 36, 45,
+				1, 10, 19, 28, 37, 46,
+				2, 11, 20, 29, 38, 47,
+				3, 12, 21, 30, 39, 48,
+				4, 13, 22, 31, 40, 49,
+				5, 14, 23, 32, 41, 50,
+				6, 15, 24, 33, 42, 51,
+				7, 16, 25, 34, 43, 52,
+				8, 17, 26, 35, 44, 53 };
+			//spotless:on
```

Comment:
Manual spotless.

```suggestion
            // spotless:off
            FILTER_MAP = new int[] {
                0,  9, 18, 27, 36, 45,
                1, 10, 19, 28, 37, 46,
                2, 11, 20, 29, 38, 47,
                3, 12, 21, 30, 39, 48,
                4, 13, 22, 31, 40, 49,
                5, 14, 23, 32, 41, 50,
                6, 15, 24, 33, 42, 51,
                7, 16, 25, 34, 43, 52,
                8, 17, 26, 35, 44, 53 };
            //spotless:on
```

Reply:
Sorry, I blame IntelliJ Scala Plugin for this one... made my tab settings wonky.